models/staging/graph/base/base_exposure_relationships.sql

```diff
@@ -0,0 +1,21 @@
+
+
+{{
+    config(
+        materialized='table',
+        post_hook="{{ generate_insert_statements_post_hook(this, resource_type='exposures', relationships=True) }}"
+    )
+}}
+
+/* Bigquery won't let us `where` without `from` so we use this workaround */
```

Comment:
do we need to add this comment to all relevant models?

Reply:
are you saying you do or do not want it in each model?

Reply:
if it's relevant for each model then yes